src/components/savingsAccounts/IRA.tsx

```diff
@@ -0,0 +1,300 @@
+  federalIncomeTaxWithheld: Number(formData.federalIncomeTaxWithheld),
+  planType: formData.planType,
+  // fields about contributions from form 5498
+  contributions: Number(formData.contributions),
+  rolloverContributions: Number(formData.rolloverContributions),
+  rothIraConversion: Number(formData.rothIraConversion),
+  recharacterizedContributions: Number(formData.recharacterizedContributions),
+  requiredMinimumDistributions: Number(formData.requiredMinimumDistributions),
+  lateContributions: Number(formData.lateContributions),
+  repayments: Number(formData.repayments)
+})
+
+const toIraUserInput = (data: Ira): IraUserInput => ({
+  ...blankUserInput,
+  ...data,
+
+  grossDistribution: data.grossDistribution.toString(),
+  taxableAmount: data.taxableAmount.toString(),
+  federalIncomeTaxWithheld: data.federalIncomeTaxWithheld.toString(),
+  // fields about contributions from form 5498
+  contributions: data.contributions.toString(),
+  rolloverContributions: data.rolloverContributions.toString(),
+  rothIraConversion: data.rothIraConversion.toString(),
+  recharacterizedContributions: data.recharacterizedContributions.toString(),
+  requiredMinimumDistributions: data.requiredMinimumDistributions.toString(),
+  lateContributions: data.lateContributions.toString(),
+  repayments: data.repayments.toString()
+})
+
+export default function IRA(): ReactElement {
+  const ira = useYearSelector(
+    (state: TaxesState) => state.information.individualRetirementArrangements
+  )
+
+  const people: Person[] = useYearSelector((state: TaxesState) => [
+    state.information.taxPayer?.primaryPerson,
+    state.information.taxPayer?.spouse
+  ])
+    .filter((p) => p !== undefined)
+    .map((p) => p as Person)
+
+  const dispatch = useYearDispatch()
+
+  const methods = useForm<IraUserInput>()
+  const { handleSubmit } = methods
+
+  const { navButtons, onAdvance } = usePager()
+
+  const onSubmitAdd = (formData: IraUserInput): void => {
+    dispatch(addIRA(toIra(formData)))
+  }
+
+  const onSubmitEdit =
+    (index: number) =>
+    (formData: IraUserInput): void => {
+      dispatch(editIRA({ index, value: toIra(formData) }))
+    }
+
+  const hsaBlock = (
+    <FormListContainer<IraUserInput>
+      items={ira.map((a) => toIraUserInput(a))}
+      onSubmitAdd={onSubmitAdd}
+      onSubmitEdit={onSubmitEdit}
+      removeItem={(i) => dispatch(removeIRA(i))}
+      icon={() => <Work />}
+      primary={(ira: IraUserInput) => ira.payer}
+      secondary={(ira: IraUserInput) => (
+        <span>
+          {IraPlanTypeTexts[ira.planType]}
+          <br />
+          gross distribution: <Currency value={toIra(ira).grossDistribution} />
+          <br />
+          contribution: <Currency value={toIra(ira).contributions} />
+          <br />
+        </span>
+      )}
+    >
+      <Grid container spacing={2}>
+        <LabeledInput
+          name="payer"
+          label="payer for this account"
```

Comment:
Minor nitpick, I think payer should be capitalized to fit the format of the rest of the form fields on the IRA page